src/cozmo/robot.py

```diff
@@ -36,7 +36,8 @@
 # __all__ should order by constants, event classes, other classes, functions.
 __all__ = ['MAX_HEAD_ANGLE', 'MIN_HEAD_ANGLE', 'MIN_LIFT_HEIGHT_MM', 'MAX_LIFT_HEIGHT_MM',
            'EvtRobotReady',
-           'GoToPose', 'DisplayOledFaceImage', 'DriveOffChargerContacts',
+           'GoToPose', 'GoToObject', 'DockWithCube', 'RollCube',
```

Comment:
These should be in alphabetical order (makes it easier to spot if something is missing)